### PR TITLE
Midnightmare Teddy game submission

### PR DESCRIPTION
**Pull request by alexthemighty**
commit of manifest and other files for submission to flathub.

### CODE REVIEW

**Comment by alexthemighty** [comment]
First time submitting an app to flathub.  Thanks for taking a look!

**Comment by nedrichards** [comment]
bot, build com.endlessnetwork.MidnightmareTeddy

File: com.endlessnetwork.MidnightmareTeddy.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <screenshot>
+      <image>https://github.com/endless-network/MidnightmareTeddy_Binary/raw/master/TeddySS4.png</image>
+      <caption>Survival or math mode</caption>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">moderate</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
```
Comment (bilelmoussaoui) [inline com.endlessnetwork.MidnightmareTeddy.appdata.xml]: You can remove the none attributes with a none value.

**Comment by bilelmoussaoui** [comment]
It would be nice if you can upstream the desktop and the appdata files.

File: com.endlessnetwork.MidnightmareTeddy.appdata.xml (hunk 1 above)
Comment (alexthemighty) [reply com.endlessnetwork.MidnightmareTeddy.appdata.xml]: Didn't know that, thanks.  The huge pile of "none" value attributes have been removed.

**Comment by alexthemighty** [comment]
> It would be nice if you can upstream the desktop and the appdata files.

Sure, I can move them up to the git repo some of the other files are being pulled from.  I had left them here with the manifest just to simplify the number of things that need editing if changes need to be made to those files.  Is there a reason it works better to have them upstream, or is that to just to simplify the eventual flathub repo?

**Comment by nedrichards** [comment]
bot, build com.endlessnetwork.MidnightmareTeddy

**Comment by nedrichards** [comment]
We like to see everything upstream since the files are useful for everyone on Linux.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.endlessnetwork.MidnightmareTeddy

**Comment by nedrichards** [comment]
@alexthemighty do you need anyone else added to the repo?

**Comment by alexthemighty** [comment]
> @alexthemighty do you need anyone else added to the repo?

It would be great if you could add nan@endlessnetwork.com with permissions on the repo as well.  Thanks!

**Comment by LeandroStanger** [comment]
FAILURE
```
/srv/buildbot-master/sandbox/lib/python3.4/site-packages/buildbot/scripts/flat-manager-client --output=output.json publish --wait https://hub.flathub.org/api/v1/build/1864
 in dir /srv/buildbot-master/master/workers/MasterWorkerUntrusted/publish/build
 watching logfiles {}
 argv: [b'/srv/buildbot-master/sandbox/lib/python3.4/site-packages/buildbot/scripts/flat-manager-client', b'--output=output.json', b'publish', b'--wait', b'https://hub.flathub.org/api/v1/build/1864']
 using PTY: True
Publishing build https://hub.flathub.org/api/v1/build/1864
Api call to https://hub.flathub.org/api/v1/build/1864/publish failed with status 400, details: {'status': 400, 'error-type': 'wrong-published-state', 'expected-state': 'published', 'current-state': 'unpublished', 'message': 'Build has already been published'}
program finished with exit code 1
elapsedTime=0.617312
```

**Comment by alexthemighty** [comment]
> FAILURE
> 
> ```
> /srv/buildbot-master/sandbox/lib/python3.4/site-packages/buildbot/scripts/flat-manager-client --output=output.json publish --wait https://hub.flathub.org/api/v1/build/1864
>  in dir /srv/buildbot-master/master/workers/MasterWorkerUntrusted/publish/build
>  watching logfiles {}
>  argv: [b'/srv/buildbot-master/sandbox/lib/python3.4/site-packages/buildbot/scripts/flat-manager-client', b'--output=output.json', b'publish', b'--wait', b'https://hub.flathub.org/api/v1/build/1864']
>  using PTY: True
> Publishing build https://hub.flathub.org/api/v1/build/1864
> Api call to https://hub.flathub.org/api/v1/build/1864/publish failed with status 400, details: {'status': 400, 'error-type': 'wrong-published-state', 'expected-state': 'published', 'current-state': 'unpublished', 'message': 'Build has already been published'}
> program finished with exit code 1
> elapsedTime=0.617312
> ```

Howdy guys.  I've noticed there is a repo for the app and a page on flathub, but it shows up as blank and I can't find the app on the site.  Is there just some processing time or is this an error that needs to be addressed?  (and if it does need to be addressed, is there something I need to do?)  Thanks!

**Comment by EndlessNaN** [comment]
@nedrichards Hi Nick! just wanted to follow up here on this thread as we are a bit unclear what the Failure status means. The [game page](https://flathub.org/apps/details/com.endlessnetwork.MidnightmareTeddy) right now appears to be empty as well. Can you help us take a look? Thank you!

**Comment by LeandroStanger** [comment]
https://flathub.org/builds/#/builders/32/builds/1956
@alexthemighty @EndlessNaN Just click on Rebuild
Captura de tela de 2019-04-03 07-28-10

**Comment by EndlessNaN** [comment]
@LeandroStanger yes we did Rebuild, however we are wondering is there anything additional we need to do to make sure that this page in flathub.org app store works: https://flathub.org/apps/details/com.endlessnetwork.MidnightmareTeddy

**Comment by LeandroStanger** [comment]
@nedrichards 
Captura de tela de 2019-04-04 07-21-33

**Pull request closed without merging**